iron-collapse.html

```diff
@@ -128,6 +128,14 @@
       return this.horizontal ? 'width' : 'height';
     },
 
+    get dimensionMax() {
```

Comment:
Could you make `dimensionMax` and `dimensionMaxCss` private properties? You could also avoid having them by using `dimension` e.g.
```javascript
/**
 * Capitalizes the first letter of the string
 * @param {!String} string
 * @private 
 */
_capitalize: function(string) {
  return string.charAt(0).toUpperCase() + string.slice(1);
},

...

  var maxDimension = "max" + this._capitalize(this.dimension);
  var maxDimensionCss = "max-" + this.dimension;